EnglishFilter/sections/specific.txt

```diff
@@ -262,8 +262,9 @@ feetporno.com,familyporn.tv,freehardcore.com,webvoyeur.com,moviesand.com,momvids
 !
 !
 downloads-anymovies.com###e15
+coin.mg##div.col-md-12:not(#go-link)
```

Comment:
Why did you add `:not(#go-link)`?

Reply:
Blocks the link in the final page.

Reply:
@zloyden

Reply:
Can you provide a screenshot with this element? I mean with this additional `id`, can't find it.

Reply:
I can't visit the site anymore. The link died. @Yuki2718 Can you bail me out?

Reply:
I can't either.

Reply:
@zloyden I guess you should just trust me, or check it out yourself.

Reply:
`coin.mg##div.col-md-12:not(#go-link)` is dangerous, and break this page
`https://coin.mg/bonus/dogecoin/?r=DG9EJ8JBUdupdoZVqcPcCN1dFr3AnrNwaH